packages/scripts/config/webpack.config.js

```diff
@@ -377,6 +382,10 @@ const scriptConfig = {
 		// generated, and the default externals set.
 		! process.env.WP_NO_EXTERNALS &&
 			new DependencyExtractionWebpackPlugin(),
+		// Removes obsolete empty scripts generated when CSS files are set as entry points.
+		new RemoveEmptyScriptsPlugin( {
+			stage: RemoveEmptyScriptsPlugin.STAGE_AFTER_PROCESS_PLUGINS,
+		} ),
```

Comment:
❤️ 
> RemoveEmptyScriptsPlugin.STAGE_AFTER_PROCESS_PLUGINS
Remove empty scripts after processing all other plugins.
For example, exact this stage needs for properly work of the @wordpress/dependency-extraction-webpack-plugin.